cassandra-analytics-integration-framework/src/main/java/org/apache/cassandra/distributed/impl/CassandraCluster.java

```diff
@@ -43,6 +43,7 @@
 import org.apache.cassandra.distributed.api.TokenSupplier;
 import org.apache.cassandra.distributed.shared.ClusterUtils;
 import org.apache.cassandra.distributed.shared.Versions;
+import org.apache.cassandra.sidecar.adapters.base.Partitioner;
```

Comment:
why does it depends on sidecar's partitioner? Can it implement with `org.apache.cassandra.spark.data.partitioner.Partitioner` instead?

It is unrelated with this patch. So I am fine w/o addressing this.

Reply:
ah yes, we don't have access to `org.apache.cassandra.spark.data.partitioner.Partitioner` here. I will not address it here, but it's something we need to address at some point